src/utils/check.rs

```diff
@@ -8,11 +9,14 @@ pub fn binary_present<T: AsRef<OsStr>>(name: T) -> bool {
 }
 
 /// Returns if ALL binaries in the list are present in the $PATH.
+#[instrument]
 pub fn binaries_present<T, L>(names: L) -> bool
```

Comment:
This could be simplified to `(names: &[impl AsRef<OsStr>])`

Reply:
Changed in df0076f.